onnxruntime/core/providers/webgpu/program.h

```diff
@@ -348,6 +351,8 @@ class ProgramBase {
   inline uint32_t DispatchGroupSizeX() const { return dispatch_group_size_x_; }
   inline uint32_t DispatchGroupSizeY() const { return dispatch_group_size_y_; }
   inline uint32_t DispatchGroupSizeZ() const { return dispatch_group_size_z_; }
+  inline const Tensor* IndirectDispatchTensor() const { return indirect_dispatch_tensor_; }
+  inline bool UseIndirectDispatch() const { return indirect_dispatch_tensor_ != nullptr; }
```

Comment:
it is not necessary to have `UseIndirectDispatch`. this should be able to determine by `IndirectDispatchTensor() == nullptr`